tests/integ/modin/frame/test_iloc.py

```diff
@@ -2710,15 +2710,15 @@ def test_df_iloc_set_with_multi_index(
         native_items.columns = pd.MultiIndex.from_tuples(item_columns)
 
     if row_key_index:
-        snow_row_key = pd.Series(row_key, index=pd.Index(row_key_index))
-        native_row_key = native_pd.Series(row_key, index=pd.Index(row_key_index))
+        snow_row_key = pd.Series(row_key, index=native_pd.Index(row_key_index))
```

Comment:
Do these tests no longer work if we pass a lazy index object?

Reply:
yeah, because we don't support a lazy MultiIndex object yet